src/java/org/apache/cassandra/service/accord/AccordSafeCommandStore.java

```diff
@@ -178,6 +178,7 @@ public RangesForEpoch ranges()
     @Override
     public void registerHistoricalTransactions(Deps deps)
     {
+        if (deps.isEmpty()) return;
```

Comment:
Should we also add an ideantical change to in memory safe store? Easier to track this way, too.

Reply:
that would require an accord patch and was trying to avoid =D

Reply:
this isn't a bug fix, its more "no-op fast"